mirrulations-core/tests/test_path_generator.py

```diff
@@ -0,0 +1,74 @@
+from mirrcore.path_generator import PathGenerator
+
+def test_get_docket_path():
+    job = {
+        "data": {
+            "id": "VETS-2005-0001",
+            "type": "dockets"
+        }  
+    } 
+    expected_path = "data/VETS/2005/VETS-2005-0001/text-VETS-2005-0001/docket/"
+    assert expected_path == PathGenerator().get_docket_path(job)
```

Comment:
`PathGenerator()` is used in all tests.  Make a `pytest` fixture to eliminate the code duplication.